### PR TITLE
zh-cn: clarify the support of PWA on Safari

### PR DESCRIPTION
**Pull request by yin1999**
### Description

clarify the support of PWA on Safari

### Related issues and pull requests

Part of: mdn/content#30463.

### CODE REVIEW

File: files/zh-cn/web/progressive_web_apps/guides/making_pwas_installable/index.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
 title: 制作可安装的 PWA
 slug: Web/Progressive_web_apps/Guides/Making_PWAs_installable
+l10n:
+  sourceCommit: bd51a95ba627fe3061d061c294ff97053c3f6ed9
 ---
 
 {{PWASidebar}}
 
-PWA 的一个定义特征是它可以安装在设备上，然后以平台特定的应用程序的形式出现在用户面前，成为用户设备的一个永久功能，用户可以像启动其他任何应用程序一样直接从操作系统启动它。
+PWA 定义的方面之一就是它可以有浏览器提示以将其安装在设备上。安装后，PWA 对用户而言就是特定于平台的应用程序，是用户设备的一个永久功能，用户可以像启动其他任何应用程序一样直接从操作系统启动它。
```
Comment (yin1999) [inline files/zh-cn/web/progressive_web_apps/guides/making_pwas_installable/index.md:10]:
```suggestion
PWA 定义的一个方面就是它可以由浏览器提示以将其安装在设备上。安装后，PWA 对用户而言就是特定于平台的应用程序，是用户设备的一个永久功能，用户可以像启动其他任何应用程序一样直接从操作系统启动它。
```

File: files/zh-cn/web/progressive_web_apps/guides/making_pwas_installable/index.md [hunk 2]
```diff
@@ -89,7 +91,7 @@ web 应用清单是一个 JSON 文件，它告诉浏览器 PWA 应该如何在
 
 ## 通过 Web 进行安装
 
-当用户访问浏览器确定可安装的网页时，用户将有机会安装它。这意味着你可以将你的 PWA 作为一个网站进行分发，使其可以通过 Web 搜索找到，也可以在应用商店中分发，以便用户可以在那里找到它。
+当支持的浏览器确定 Web 应用程序满足前述的可安装性标准时，它将提示用户安装当前应用。这意味着你可以将你的 PWA 作为一个网站进行分发，使其可以通过 Web 搜索找到，也可以在应用商店中分发，以便用户可以在那里找到它。
```
Comment (yin1999) [inline files/zh-cn/web/progressive_web_apps/guides/making_pwas_installable/index.md:94]:
```suggestion
当支持的浏览器确定 Web 应用程序满足前述的可安装性要求时，它将提示用户安装当前应用。用户将有机会安装该应用程序。这意味着你可以将你的 PWA 通过网站进行分发，使其可以通过 Web 搜索找到，也可以在应用商店中分发，以便用户可以在那里找到它。
```

File: files/zh-cn/web/progressive_web_apps/guides/making_pwas_installable/index.md [hunk 3]
```diff
@@ -103,21 +105,23 @@ web 应用清单是一个 JSON 文件，它告诉浏览器 PWA 应该如何在
...
 
 - 在 Android、Firefox、Chrome、Edge、Opera 和 Samsung Internet Browser 上都支持安装 PWA。
-
 - 在 iOS 16.3 及更早版本上，PWA 只能通过 Safari 安装。
-
 - 在 iOS 16.4 及更高版本上，PWA 可以通过 Safari、Chrome、Edge、Firefox 和 Orion 的分享菜单安装。
 
+### 将网站安装为应用程序
+
+桌面版和移动版 Safari 以及桌面版 Edge 也支持将任何网站安装为应用程序。但是，这并不是 PWA 所特有的，因为网站不需要满足本指南中描述的可安装性标准，且浏览器不会主动提示安装网站。
```
Comment (yin1999) [inline files/zh-cn/web/progressive_web_apps/guides/making_pwas_installable/index.md:123]:
```suggestion
桌面版和移动版 Safari 以及桌面版 Edge 也支持将任何网站安装为应用程序。但是，这并不是 PWA 所特有的，因为网站不需要满足本指南中描述的可安装性要求，且浏览器不会主动提示安装网站。
```

File: files/zh-cn/web/progressive_web_apps/guides/making_pwas_installable/index.md [hunk 4]
```diff
@@ -1,24 +1,26 @@
...
 title: 制作可安装的 PWA
 slug: Web/Progressive_web_apps/Guides/Making_PWAs_installable
+l10n:
+  sourceCommit: bd51a95ba627fe3061d061c294ff97053c3f6ed9
 ---
 
 {{PWASidebar}}
 
-PWA 的一个定义特征是它可以安装在设备上，然后以平台特定的应用程序的形式出现在用户面前，成为用户设备的一个永久功能，用户可以像启动其他任何应用程序一样直接从操作系统启动它。
+PWA 定义的一个方面就是它可以由浏览器提示以将其安装在设备上。安装后，PWA 对用户而言就是特定于平台的应用程序，是用户设备的一个永久功能，用户可以像启动其他任何应用程序一样直接从操作系统启动它。
```
Comment (jasonren0403) [inline files/zh-cn/web/progressive_web_apps/guides/making_pwas_installable/index.md:10]:
```suggestion
PWA 定义的一个方面就是它可以由浏览器提示以将其安装在设备上。安装后，PWA 对用户而言就是特定于平台的应用程序，是用户设备的一个永久特性，用户可以像启动其他任何应用程序一样直接从操作系统启动它。
```

**Review by jasonren0403 — APPROVED** [review]
LGTM, with one minor nit 👍

**Pull request merged**